modules/apps/depot/depot-web/src/main/java/com/liferay/depot/web/internal/portlet/action/EditDepotEntryMVCActionCommand.java

```diff
@@ -120,6 +120,14 @@ private void _updateDLSizeLimitConfiguration(
 
 		Map<String, String[]> parameterMap = actionRequest.getParameterMap();
 
+		Long fileMaxSize = 0L;
+
+		String[] inputSize = parameterMap.get("fileMaxSize");
+
+		if ((inputSize.length != 0) && Validator.isNotNull(inputSize[0])) {
+			fileMaxSize = GetterUtil.getLong(inputSize[0]);
+		}
+
```

Comment:
Don't use wrapper types unless strictly necessary (Long vs long).

Also, can you use `MapUtil` to get the value and get rid of the `inputSize` variable? `MapUtil.getLong(..., "fileMaxSize", 0)`.

Reply:
Changed, I was trying to unify it with the rest of the method, but completely agree that it is a better solution.